envoy/upstream/upstream.h

```diff
@@ -1001,11 +1001,7 @@ class ClusterInfo : public Http::FilterChainFactory {
   virtual OptRef<const LoadBalancerConfig> loadBalancerConfig() const PURE;
 
   /**
-   * @return the load balancer factory for this cluster if the load balancing type is
-   * LOAD_BALANCING_POLICY_CONFIG.
-   * TODO(wbpcode): change the return type to return a reference after
-   * 'envoy_reloadable_features_convert_legacy_lb_config' is removed. The factory should never be
-   * nullptr when the load balancing type is LOAD_BALANCING_POLICY_CONFIG.
+   * @return the load balancer factory for this cluster.
```

Comment:
if you're not doing the TODO re reference, comment about null returns?

Reply:
will do that, no null return is allowed now. Will add the TODO back.